hummingbot/paradex_perpetual/paradex_perpetual_constants.py

```diff
@@ -0,0 +1,100 @@
+from hummingbot.core.api_throttler.data_types import LinkedLimitWeightPair, RateLimit
+from hummingbot.core.data_type.in_flight_order import OrderState
+
+EXCHANGE_NAME = "paradex_perpetual"
+BROKER_ID = "HBOT"
+MAX_ORDER_ID_LEN = None
+
+MARKET_ORDER_SLIPPAGE = 0.05
+
+DOMAIN = EXCHANGE_NAME
+TESTNET_DOMAIN = "paradex_perpetual_testnet"
+
+PROD_BASE_URL = "https://api.prod.paradex.trade/v1"
+TESTNET_BASE_URL = "https://api.testnet.paradex.trade/v1"
+
+PROD_WS_URL = "wss://ws.api.prod.paradex.trade/v1"
+TESTNET_WS_URL = "wss://ws.api.testnet.paradex.trade/v1"
```

Comment:
Small suggestion for the future. These URL should be built in general using the env we are in. Code shouldn't be selecting later on.

If we could do this in a central place and reduce duplication.